CHANGES.md

```diff
@@ -57,7 +57,7 @@ New languages:
   none.
 
 New styles:
-  none.
+  -*Gradient Dark* by [Samia Ali][]
```

Comment:
This needs to be under 9.17 not 9.16... and you also need to add the link to the footnotes (see the other examples).

Reply:
Needs a space after `-` for bulleted list.

Reply:
Thank you! I wasn't sure which version to put it under

Reply:
Waiting for you to push an update.